docs/docs/recipes/deploying-your-site.md

```diff
@@ -103,7 +103,7 @@ Use [Now CLI](https://zeit.co/download) to deploy your Gatsby application withou
 
 ### Additional resources
 
-- [Deploying to ZEIT Now](/docs/deploying-to-zeit-now/)
+- [Deploying to Vercel Now](/docs/deploying-to-now/)
```

Comment:
```suggestion
- [Deploying to Vercel](/docs/deploying-to-vercel/)
```

Reply:
also more `now` commands on this page should be `vercel` as the timeline shows the names of cli also change